tests/test_update_storage_class.py

```diff
@@ -52,6 +52,7 @@ def test_update(self) -> None:
         # Register few dataset types. For this test we want to use storage
         # classes that do not require setting up additional packages.
         butler = Butler.from_config(butler_root, writeable=True)
+        self.enterContext(butler)
```

Comment:
I think you can do `with Butler.from_config()` here too, so that butler gets destroyed at the exit from context. But this is fine too.

Reply:
Minimizing updates is fine, let's keep it 👍